staticd/static_main.c

```diff
@@ -170,6 +170,10 @@ FRR_DAEMON_INFO(staticd, STATIC, .vty_port = STATIC_VTY_PORT,
 
 		.privs = &static_privs, .yang_modules = staticd_yang_modules,
 		.n_yang_modules = array_size(staticd_yang_modules),
+
+		// avoid reading split config, mgmtd will do this for us now
```

Comment:
please use c style comments

Reply:
For some reason C-style comment got flagged as wrong in my editor, I was wondering if it was interacting poorly with the macro for some reason. I can move the comment above I guess.